src/app/Marine2/utils/media.tsx

```diff
@@ -0,0 +1,39 @@
+export type StylesType = {
+  [key: string]: {
+    [key: string]: string
+  }
+}
+
+type ComponentSizeType = {
+  width: number
+  height: number
+}
+
+type BreakpointsType = {
+  [key: string]: ComponentSizeType
+}
+
+const boxBreakpoints: BreakpointsType = {
+  "sm-s": { width: 200, height: 200 },
+  "sm-m": { width: 200, height: 450 },
+  "sm-l": { width: 200, height: 600 },
+  "md-s": { width: 400, height: 200 },
+  "md-m": { width: 400, height: 450 },
+  "md-l": { width: 400, height: 600 },
+  "lg-s": { width: 600, height: 200 },
+  "lg-m": { width: 600, height: 450 },
+  "lg-l": { width: 600, height: 600 },
+}
```

Comment:
Good to define these now, definitively. That way we don't have to go back and change a bunch of styles later on. If these match up with the design, they are fine with me.

Reply:
I agree, if we are sure these breakpoints cover all cases needed